sdk/src/CrashReporter.ts

```diff
@@ -200,6 +208,48 @@ export default class CrashReporter {
       return;
     }
 
+    if (isFatal) {
+      setCurrentTags(getCurrentTags().concat('UnhandledError'))
+    }
```

Comment:
Can errors occur after a "isFatal" exception? 

If so, then I believe those errors will also have the "UnhandledError" tag applied, which would be undesired. 

Since tags are also used with RUM events, those RUM events will have this tag which, I believe, would be undesired